docs/sdk/dev/initialize-and-run-workflows.md

```diff
@@ -22,12 +25,7 @@ workflow:
 ```
 
 ---
-
-{% info_block infoBox "Project limitations" %}
-
-The project is limited to the workflows specified during initialization.
-
-{% endinfo_block %}
+**Note:** Project is limited to the workflows specified during initialization.
```

Comment:
{% info_block warningBox "Project Limits" %}

The project is limited to the workflows specified during initialization.

 {% endinfo_block %}